packages/playground/src/components/project-settings/form-member-delete-dialog.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import {
+  AlertDialog,
+  AlertDialogTrigger,
+  AlertDialogContent,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogCancel,
+  Button,
+  Icon,
+  Spacer
+} from '@harnessio/canary'
+
+interface MemberProps {
+  display_name?: string
+}
+interface FormDeleteMemberDialogProps {
+  member: MemberProps
+  onClose: () => void
+  onDelete: () => void
+  isDeleting: boolean
+  deleteSuccess: boolean
+}
+
+export const FormDeleteMemberDialog: React.FC<FormDeleteMemberDialogProps> = ({
+  member,
+  onClose,
+  onDelete,
+  isDeleting,
+  deleteSuccess
+}) => {
+  return (
+    <AlertDialog open={true} onOpenChange={onClose}>
```

Comment:
Quick question: why did we use alertdialog as opposed to dialog or popover?

Reply:
[Alert Dialog](https://ui.shadcn.com/docs/components/alert-dialog) always renders `Cancel` and `Continue` buttons, which are your primary and secondary call-to-actions, also ensuring proper positioning of these buttons, which is something `Dialog` doesn't do.